Assets/Scripts/Configuration/Configurator.cs

```diff
@@ -31,9 +31,13 @@ public class Configurator : ScriptableObject, IWindow
         private Uri uri;
 
         [SerializeField] 
-        [Tooltip("The scene with the Setup Wizard that needs to load additively")]
+        [Tooltip("The scene with the Setup Window that needs to load additively")]
         private string setupSceneName;
 
+        [SerializeField]
+        [Tooltip("The main scene that will be loaded after the setup scene is closed")]
+        private string mainSceneName;
```

Comment:
is het wel handig om hier een string voor te gebruiken?

Reply:
Je kan Scene helaas niet serializen omdat die eigenlijk geen runtime 'Assets' zijn. Dus kan geen directe reference naar de Scene asset gebruiken; dus is dit de 'makkelijkste' oplossing